onnxscript/rewriter/llama_rule_sets.py

```diff
@@ -76,8 +136,39 @@ def rewrite(cls, op, x: ir.Value, perm1: ir.Attr, perm2: ir.Attr):
         return op.Transpose(x, perm=last)
 
 
+class UnsqueezeUnsqueeze(orp.RewriteRuleAsClass):
+    """Replaces ``Unsqueeze(Unsqueeze(., axes1), axes2)``
+    with one Unsqueeze.
+    """
+
+    @classmethod
+    def pattern(cls, op, x, axes1, axes2):
+        return op.Unsqueeze(op.Unsqueeze(x, axes1), axes2)
+
+    @classmethod
+    def _combine1(cls, axes1: np.ndarray, axes2: np.ndarray) -> np.ndarray:
```

Comment:
Does this handle negative values as axes?

Reply:
No. I'll remove that as well.